content/english/linux-basics/activity-8.md

```diff
@@ -7,6 +7,8 @@ difficulty: "Intermediate"
 weight: 11
```

Comment:
Remove the period at the end of the description and prereqs for all files.

Reply:
Should I get rid of difficulty as well?